packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx

```diff
@@ -46,6 +46,16 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
       })
       .replace(/\d/g, "")
       .trim();
+
+  useEffect(() => {
+    if (!getAppData("currency")) {
+      setAppData("currency", currencyOptions[0].value);
+    }
+    if (!getAppData("paymentOption")) {
+      setAppData("paymentOption", paymentOptions[0].value);
+    }
+  }, []);
```

Comment:
When mounting event type app card, we set initial values that will only be saved after submit.